File: expresslk-services/ExpressLK-core/src/main/java/lk/express/AvailabilityCriteria.java
package lk.express;

import javax.xml.bind.annotation.XmlRootElement;
import javax.xml.bind.annotation.XmlType;

@XmlRootElement
@XmlType(name = "AvailabilityCriteria", namespace = "http://express.lk")
public class AvailabilityCriteria {

	private String resultIndex;
	private int sectorIndex;

	private int scheduleId;
	private int boardingLocationId;
	private int droppingLocationId;
	private int busTypeId;

	public int getScheduleId() {
		return scheduleId;
	}

	public void setScheduleId(int scheduleId) {
		this.scheduleId = scheduleId;
	}

	public int getBoardingLocationId() {
		return boardingLocationId;
	}

	public void setBoardingLocationId(int boardingLocationId) {
		this.boardingLocationId = boardingLocationId;
	}

	public int getDroppingLocationId() {
		return droppingLocationId;
	}

	public void setDroppingLocationId(int droppingLocationId) {
		this.droppingLocationId = droppingLocationId;
	}

	public int getBusTypeId() {
		return busTypeId;
	}

	public void setBusTypeId(int busTypeId) {
		this.busTypeId = busTypeId;
	}

	public String getResultIndex() {
		return resultIndex;
	}

	public void setResultIndex(String resultIndex) {
		this.resultIndex = resultIndex;
	}

	public int getSectorIndex() {
		return sectorIndex;
	}

	public void setSectorIndex(int sectorIndex) {
		this.sectorIndex = sectorIndex;
	}
}

File: expresslk-webportal/module/Api/src/Api/Client/Soap/Core/AvailabilityCriteria.php
<?php

namespace Api\Client\Soap\Core;

class AvailabilityCriteria
{

    /**
     * @var int $boardingLocationId
     * @access public
     */
    public $boardingLocationId = null;

    /**
     * @var int $busTypeId
     * @access public
     */
    public $busTypeId = null;

    /**
     * @var int $droppingLocationId
     * @access public
     */
    public $droppingLocationId = null;

    /**
     * @var string $resultIndex
     * @access public
     */
    public $resultIndex = null;

    /**
     * @var int $scheduleId
     * @access public
     */
    public $scheduleId = null;

    /**
     * @var int $sectorIndex
     * @access public
     */
    public $sectorIndex = null;

    /**
     * @param int $boardingLocationId
     * @param int $busTypeId
     * @param int $droppingLocationId
     * @param string $resultIndex
     * @param int $scheduleId
     * @param int $sectorIndex
     * @access public
     */
    public function __construct($boardingLocationId, $busTypeId, $droppingLocationId, $resultIndex, $scheduleId, $sectorIndex)
    {
      $this->boardingLocationId = $boardingLocationId;
      $this->busTypeId = $busTypeId;
      $this->droppingLocationId = $droppingLocationId;
      $this->resultIndex = $resultIndex;
      $this->scheduleId = $scheduleId;
      $this->sectorIndex = $sectorIndex;
    }

}
